src/main/java/org/sagebionetworks/web/client/widget/entity/browse/EntityFinderV2ViewImpl.java

```diff
@@ -231,7 +224,8 @@ private static native void _showEntityFinderReactComponent(Element el, String se
 			};
 			$wnd.ReactDOM.render(
 				$wnd.React.createElement($wnd.SRC.SynapseComponents.EntityFinder, props, null),
-				el
+				el,
+				function() { onRender.@java.lang.Runnable::run()() }
```

Comment:
What do you think about adding a "react component integration" section to the SWC readme that would contain generally useful solutions like this?